_posts/2020-02-09-single-responsibility-principle.md

```diff
@@ -0,0 +1,271 @@
+
+This article explains the Single Responsibility Principle (SRP): what does it practically mean, when and how to apply it.
+
+ - [What does the Single Responsibility Principle say?](#what-does-the-single-responsibility-principle-say)
```

Comment:
We don't need a table of contents, it's generated automatically in the side bar.

Reply:
Removed